plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/BasePostgresFailureRecoveryTest.java

```diff
@@ -81,4 +81,20 @@ protected void testUpdate()
                 .withCleanupQuery(cleanupQuery)
                 .isCoordinatorOnly();
     }
+
+    @Test
+    @Override
+    protected void testDeleteWithSubquery()
+    {
+        // TODO: support merge with fte
+        abort("skipped");
+    }
+
+    @Test
+    @Override
+    protected void testMerge()
+    {
+        // TODO: support merge with fte
+        abort("skipped");
```

Comment:
please assert the failure of the test. I would like to see that it truly fails because of fte